Master

akrap19 opened the pull request.

Commit:
Možda...

ncaklovic commented:
Ima još nešto posla:
- dialog za broj uvijek pokazuje 10 (bez obzira na trenutni broj)
- inicijalni font nema ništa postavljeno. U konstruktoru MainWindow postavite barem lfFaceName i lfHeight. Trebat će vam formula sa predavanja (da biste postavili veličinu u pointima) i HDC (uzmite GetDC(0), ne zaboravite pozvati ReleaseDC)
- promjena fonta ne iscrta ponovno prozor
- Cancel na font dialogu svejedno promijeni font
- da li podržavate boju fonta/teksta? Ako ne, maknite sa font dialoga (pomoću font.Flags).

akrap19 committed:
Update main.cpp

Riješio sam sve greške  osim greške sa cancelom u izboru fonta, ne mogu uopče shvatiti u čemu je problem.

akrap19 committed:
Update main.h

ncaklovic commented:
Skoro sve popravljeno, ali:
- formula za lfHeight je kriva, pogledajte još jednom predavanja (ili [MSDN](https://msdn.microsoft.com/ja-jp/library/ms931108.aspx))
- font se promijeni jer u ChooseFont funkciju šaljete direktno pointer na svoj LOGFONT (&lf) pa ga uvijek promijeni. Da bi podržavali Cancel pošaljite kopiju i tek ako funkcija vrati true iskopirajte LOGFONT strukturu u svoj član.

akrap19 committed:
Update main.cpp

akrap19 committed:
Update main.h

ncaklovic commented:
Pa niste ništa promijenili što se tiče fonta, i dalje šaljete pointer. Samo ste odgodili crtanje sa promijenjenim fontom (promijenite veličinu prozora nakon što odaberete Cancel u font dialogu).
Usput, možete više fileova uključiti u jedan commit (ne morate zasebno .h pa .cpp) - i dati mu smisleniji komentar.

akrap19 committed:
Update main.cpp

ncaklovic commented:
Sve u redu.
